package.json

```diff
@@ -62,7 +62,7 @@
     "@babel/preset-env": "^7.7.1",
     "@babel/preset-react": "^7.7.0",
     "@storybook/react": "^6.5.0-beta.8",
-    "@stripe/stripe-js": "^1.35.0",
+    "@stripe/stripe-js": "^1.36.0",
```

Comment:
Can the peer dependencies below also be bumped?

Reply:
Yes, done